projects/ids-enterprise-ng/src/lib/weekview/soho-weekview.component.ts

```diff
@@ -0,0 +1,507 @@
+/// <reference path="soho-weekview.d.ts" />
+
+import {
+  AfterViewChecked,
+  AfterViewInit,
+  ChangeDetectorRef,
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  EventEmitter,
+  HostBinding,
+  Input,
+  NgZone,
+  OnDestroy,
+  Output,
+} from '@angular/core';
+
+/********************************************************************
+ * Weekview component
+ *******************************************************************/
+@Component({
+  selector: 'div[soho-weekview]', // tslint:disable-line
```

Comment:
Should we call it `soho-week-view` to be consistent? I did this on purpose in the underlying component because we did it wrong by not hashing the names with `-` before and it's causing ES6 need to be shived in a strange way.

So following the same name would be good.

Reply:
Changing to soho-week-view